### ISSUE TITLE
GG Max - EXP Build - DPI awareness not working for the HUD ?

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#4328]
HI, I Have just download the latest version of the EXP Max.
When i test my level, the HUD is now really small to what it was before ?
Anyone else having these issues ?

I am using a 4k monitor.

**Comment by SampeyFS** [comment]
![image](https://user-images.githubusercontent.com/108415264/227996888-a75726b7-9b67-4f1e-940d-b7509a6ba54b.png)

**Comment by SampeyFS** [comment]
yes too small so you need monocle to see ahahahh better first  :)

**Comment by Tum1370** [comment]
I wonder if this issue is caused by the ongoing development of VR, as the HUD and everything will need adding in to VR.

**Comment by LukeTheLukeLTL** [comment]
They fixed the problem after the last update.
![BugFixed](https://user-images.githubusercontent.com/103126963/228318203-e4a4e492-99e8-4f59-a74b-c93aef3b1606.png)

**Comment by Tum1370** [comment]
Ye just tested latest update and can confirm this bug is fixed. 
Closing the issue now.
